### ISSUE TITLE
Error installing npm install -g

### ISSUE THREAD

**Issue by natew** [wistityhq/strapi#122]
```
npm install strapi -g                                                                                     relations ✭ ✖ ✱ ◼
npm WARN deprecated koa-gzip@0.1.0: please use koa-compress instead.
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/nw/n/bin/node" "/Users/nw/n/bin/npm" "install" "strapi" "-g"
npm ERR! node v5.10.1
npm ERR! npm  v3.8.3
npm ERR! code EREADFILE

npm ERR! Error extracting /Users/nw/.npm/sails-disk/0.10.8/package.tgz archive: ENOENT: no such file or directory, open '/Users/nw/.npm/sails-disk/0.10.8/package.tgz'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/nw/company/triangle/npm-debug.log
```

When i `cd /Users/nw/.npm/sails-disk` and ls:

```
0.10.9
```

Looking for 0.10.8, but theres 0.10.9.

**Comment by pierreburgy** [comment]
Hi @natew,

Is this error due to Strapi or sails-disk?

Did you solved the issue?

**Comment by natew** [comment]
Wasn't solved but not using strapi anymore

On Thursday, May 26, 2016, Pierre BURGY <notifications@github.com> wrote:

> Closed #122 <https://github.com/wistityhq/strapi/issues/122>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/wistityhq/strapi/issues/122#event-673312939>
>


-- 
natewienert.com @natebirdman <http://twitter.com/natebirdman>
